### PR TITLE
Spruce up Timeline UI

### LINKED ISSUE #58: Spruce up Admin Timeline View
**Is your feature request related to a problem? Please describe.**
The Timeline View for all objects in the Admin has been reported as confusing.

**Describe the solution you'd like**
Investigate and come up with a better solution for displaying past and upcoming changes in this view.

**Additional context**
v3-edge demo workarea com_admin_timeline_Z2lkOi8vZGVtby9Xb3JrYXJlYTo6Q29udGVudC81YzU0ODhkNDhlNDRkZDUwMGIyZDNjMGQ

### PR DESCRIPTION
**Pull request by meowsus**
The `activity`, `activity-group`, and `date-marker` UIs couple together
to create, what's unofficially referred to as, The Timeline UI. These
components have been neglected for a long time... until now!

Closes #58

### CODE REVIEW

**Comment by meowsus** [comment]
Screenshot from 2019-09-25 12-47-12

File: admin/app/views/workarea/admin/activities/show.html.haml [hunk 1]
```diff
@@ -24,20 +24,27 @@
...
-              - else
-                = relative_weekday(day)
-          .activity-group__activities
-            - entries.each do |entry|
-              = render_activity_entry(entry)
+          .activity-group
+            .grid
+              .grid__cell.grid__cell--20
+                .date-marker
+                  - if day == Date.today
```
Comment (bencrouse) [inline admin/app/views/workarea/admin/activities/show.html.haml]: A Railsier way to write this is `day.today?`

Comment (meowsus) [reply admin/app/views/workarea/admin/activities/show.html.haml]: I nabbed this logic from some other file, but after I do another quick audit of use-cases I can see what I can do about cleaning these up.

**Comment by meowsus** [comment]
@bencrouse the timeline looks to be working for me. If anything is bonkers with it in a more heavily tested environment, I am sure @ebigler will sniff it out.

localhost_3000_admin_timeline_Z2lkOi8vZHVtbXktYXBwL1dvcmthcmVhOjpDYXRhbG9nOjpDYXRlZ29yeS81ZDhhNzAwYWQ0MzIyYTUyNTJhMjZjM2U

**Comment by bencrouse** [comment]
@meowsus but not chronological for upcoming changes, I reproduced, check out the image.
Screen Shot 2019-09-26 at 1 04 29 PM

**Comment by meowsus** [comment]
Gimmie a minute here, dudes. I've got to clear #124 before this can be merged.

**Comment by bencrouse** [comment]
Holding off merge for #124

**Pull request merged**